pointers/task_2.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#define MAXN 100
+#define MAXM 100
+using namespace std;
+int main(){
+    int n, m, a[MAXN], b[MAXM], counter = 0;
+    bool visited[MAXN] = {0};
+    cin >> n;
+    for(int i = 0; i < n; i++){
+        cin >> *(a + i);
+    }
+    cin >> m;
+    for(int i = 0; i < m; i++){
+        cin >> *(b + i);
+    }
+
+    for(int i = 0; i < n; i++){
+        for(int j = 0; j < m; j++){
+            if((*(a + i) == *(b + j)) && i < j){
+                counter++;
+            }
+        }
```

Comment:
По-добре е директно да се спести допълнителното въртене на цикъла. 
```suggestion
        for(int j = i+1; j < m; j++){
            if((*(a + i) == *(b + j)){
                counter++;
            }
        }
```